lib/netsuite.rb

```diff
@@ -9,7 +9,13 @@
 module NetSuite
   autoload :Configuration,    'netsuite/configuration'
   autoload :Response,         'netsuite/response'
-  autoload :AsyncStatus,      'netsuite/async_status'
+  autoload :Status,           'netsuite/status'
+
+  module Async
+    autoload :AsyncStatus,        'netsuite/async/async_status'
```

Comment:
mhh, I guess since the SOAP object is AsyncStatus you're keeping that then.
If so, why nest it ?